### PR TITLE
ci: add a GitHub workflow to test a module with CITGM

### PR DESCRIPTION
**Pull request by targos**
Form:

image

Example run on my fork with `lodash` module: https://github.com/targos/citgm/actions/runs/623994089

Another example with:
- module: https-proxy-agent
- repository: lpinca/citgm
- ref: add/https-proxy-agent
https://github.com/targos/citgm/actions/runs/624006145

### CODE REVIEW

**Review by richardlau — APPROVED** [review]
Is there an at-a-glance way to see the parameters that were used to start the runs?

**Comment by targos** [comment]
> Is there an at-a-glance way to see the parameters that were used to start the runs?

I haven't found anything in the GitHub UI.
We could hack something with a no-op job whose name contains the parameters. WDYT?

**Comment by richardlau** [comment]
> > Is there an at-a-glance way to see the parameters that were used to start the runs?
> 
> I haven't found anything in the GitHub UI.
> We could hack something with a no-op job whose name contains the parameters. WDYT?

SGTM.

**Comment by targos** [comment]
https://github.com/targos/citgm/actions/runs/624243277

image

image

**Review by richardlau — APPROVED** [review]

File: .github/workflows/test-module.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Log workflow parameters
+        env:
+          WORKFLOW_PARAMETERS: ${{ toJSON(github.event.inputs) }}
+        run: echo "$WORKFLOW_PARAMETERS"
+
+  test-module:
+    name: Test on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
```
Comment (ljharb) [inline .github/workflows/test-module.yml:31]: if you omit this name, then the list should automatically show the matrix parameters in parens

**Review by richardlau — APPROVED** [review]

**Comment by targos** [comment]
Open https://github.com/nodejs/citgm/actions/workflows/test-module.yml to start a run.

**Pull request merged**